--- origsrc/vcdimager-0.7.23/frontends/cli/Makefile.am	2004-02-11 19:57:23.000000000 -0600
+++ src/vcdimager-0.7.23/frontends/cli/Makefile.am	2007-02-16 00:23:43.234375000 -0600
@@ -28,7 +28,7 @@
 cdxa2mpeg_LDADD =  $(LIBISO9660_LIBS) $(LIBVCD_LIBS) $(LIBPOPT_LIBS)
 
 vcd_info_SOURCES = vcd-info.c
-vcd_info_LDADD = $(LIBISO9660_LIBS) $(LIBVCDINFO_LIBS) $(LIBVCD_LIBS) $(LIBPOPT_LIBS) $(LIBCDIO_LIBS) $(LIBISO9660_LIBS)
+vcd_info_LDADD = $(LIBISO9660_LIBS) $(LIBVCDINFO_LIBS) $(LIBPOPT_LIBS) $(LIBCDIO_LIBS) $(LIBISO9660_LIBS)
 
 man_MANS = vcdimager.1 cdxa2mpeg.1 vcd-info.1
 
--- origsrc/vcdimager-0.7.23/frontends/xml/Makefile.am	2004-08-15 07:16:49.000000000 -0500
+++ src/vcdimager-0.7.23/frontends/xml/Makefile.am	2007-02-16 00:23:43.250000000 -0600
@@ -50,7 +50,7 @@
 	vcd_xml_parse.c \
 	vcd_xml_parse.h 
 
-vcdxrip_LDADD = $(XML_LIBS) $(LIBISO9660_LIBS) $(LIBVCDINFO_LIBS) $(LIBVCD_LIBS) $(LIBPOPT_LIBS) $(LIBCDIO_LIBS)
+vcdxrip_LDADD = $(XML_LIBS) $(LIBISO9660_LIBS) $(LIBVCDINFO_LIBS) $(LIBPOPT_LIBS) $(LIBCDIO_LIBS)
 vcdxrip_SOURCES = \
 	vcd_xml_rip.c \
 	vcd_xml_common.c \
--- origsrc/vcdimager-0.7.23/lib/Makefile.am	2005-07-10 07:20:03.000000000 -0500
+++ src/vcdimager-0.7.23/lib/Makefile.am	2007-02-16 00:23:43.250000000 -0600
@@ -97,7 +97,7 @@
 	vcd_read.h 
 
 libvcdinfo_la_LIBADD = $(LIBCDIO_LIBS) $(LIBISO9660_LIBS) $(LIBVCD_LIBS)
-libvcdinfo_la_LDFLAGS = -version-info $(libvcdinfo_la_CURRENT):$(libvcdinfo_la_REVISION):$(libvcdinfo_la_AGE)
+libvcdinfo_la_LDFLAGS = -version-info $(libvcdinfo_la_CURRENT):$(libvcdinfo_la_REVISION):$(libvcdinfo_la_AGE) -no-undefined
 libvcdinfo_la_DEPENDENCIES = libvcd.la
 
 # Below is some simple versioning where every symbol is versioned with
